packages/trader/src/Modules/SmartChart/Components/Markers/marker.jsx

```diff
@@ -25,11 +25,12 @@ const ChartMarker = ({ marker_config, marker_content_props, is_bottom_widget_vis
         }
     };
 
-    return (
-        <FastMarker markerRef={onRef}>
-            <ContentComponent {...toJS(marker_content_props)} />
-        </FastMarker>
-    );
+    // memoizing the marker components data:
+    const getMemoizedComponent = React.useCallback(() => {
+        return <ContentComponent {...toJS(marker_content_props)} />;
```

Comment:
I think this won't update `marker_content_props` unless we add a dependency of it.